checker/tests/run-pass/post_conditions.rs

```diff
@@ -24,12 +24,12 @@ pub fn foo(bl: Block) {
     verify!(ret < std::u64::MAX - 1);
 }
 
-pub fn bad(c: bool) -> u64 {
+pub fn bar(c: bool) -> u64 {
     let result = result!();
     if c {
         assumed_postcondition!(result < 5);
     } else {
-        assumed_postcondition!(result > 10); //~ only one post condition is supported
+        assumed_postcondition!(result > 10);
     }
     result
```

Comment:
It would be great if you can now add some calls to the function and verify that the results are as expected.

Reply:
That was one question I already asked: how to test this? Can you point to an example test which involves logic like this?